tox.ini

```diff
@@ -249,3 +250,10 @@ deps =
     {[dev]deps}
     {[testenv:docs]deps}
     {[testenv:lint]deps}
+commands =
+    mypy -m nunavut \
```

Comment:
This change pulls some missing linting dependencies for the `tox -e local` target.